### DISCUSSION TITLE
How to handle backing-up Postgres?

### DISCUSSION THREAD

**Discussion by zadamg** [vercel/community#2731] [Help]
### Summary

Hi there,

As I start to collect real users in my application's postgres instance on Vercel, i'm getting anxious wondering how I manage backups.  How do I back-up my db?

Thanks!

Zack

### Example

https://vercel.com/dashboard/stores/postgres/

### Steps to Reproduce

How to replicate me, discovering said problem:

step1:  navigate to:  https://vercel.com/dashboard/stores/postgres
step2:  discover:  alas, no option to back-up my db. 
step3:  go to documentation searching for info:  alas, none. panic.
step4:  consider a "pgdump" from the "query" tab.
step5:  realize you don't know where that goes
step6:  re-panic. 🦀

**Comment by amyegan** [comment]
Hi @zadamg. Using [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) is the appropriate solution at the moment. I run backups of my example database on my local computer using `pg_dump <connection-string> --file=<my-backup-filename>.sql --file=<db-name>-$(date +%F).sql`. You can use the `POSTGRES_URL` environment variable value as the connection string or find the it near the top of the database's details page.

**Reply by ColinPeyrat** [reply]
Hi ! 
Where do you run this command ? Running it in the "Query" tab give me `Syntax error: syntax error at or near "pg_dump".`
